src/libaktualizr/package_manager/packagemanagerinterface.cc

```diff
@@ -98,6 +112,7 @@ bool PackageManagerInterface::fetchTarget(const Uptane::Target& target, Uptane::
       return true;
     }
     std::unique_ptr<DownloadMetaStruct> ds = std_::make_unique<DownloadMetaStruct>(target, progress_cb, token);
+    ds->time_start = ds->time_lastreport = std::chrono::steady_clock::now(); //use this time to report download progress it download take long time
```

Comment:
Is `time_start` used for anything? Seems unnecessary.

Reply:
removed. it was artifact from older and a bit more complicated version